tracecat/db.py

```diff
@@ -153,6 +162,22 @@ class WorkflowRun(Resource, table=True):
     action_runs: list["ActionRun"] | None = Relationship(back_populates="workflow_run")
 
 
+class WorkflowSchedule(Resource, table=True):
```

Comment:
test this with fkey constraints enabled. sqlite doesn't have this on by default

Reply:
Later